diff --git a/autosub.lua b/autosub.lua
index f399f52..7421918 100644
--- a/autosub.lua
+++ b/autosub.lua
@@ -3,7 +3,7 @@
 --=============================================================================
 --          This script uses Subliminal to download subtitles,
 --          so make sure to specify your system's Subliminal location below:
-local subliminal = '/home/david/.local/bin/subliminal'
+local subliminal = 'subliminal'
 --=============================================================================
 -->>    SUBTITLE LANGUAGE:
 --=============================================================================
